base/common/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -1516,11 +1282,9 @@ static Name getJssName(boolean enable_encoding, String dn) {
                         ret.addOrganizationalUnitName(split[1]);
                     }
                     //              System.out.println("OU found : " + split[1]);
-                    continue;
```

Comment:
It might be possible to chain the if-statements above into a series of if-else-statements so we can remove all of the `continue` statements in this loop.

Reply:
Yeah that would be much easier to read, for a future update though I think I have changed enough for one commit.